src/lib.rs

```diff
@@ -72,9 +72,11 @@ use alloc::{
 pub use ttf_parser::Language;
 pub use ttf_parser::Width as Stretch;
 
+use slotmap::DenseSlotMap;
+
 /// A unique per database face ID.
 ///
-/// We're using `u32` internally, which is auto-incremented for each face.
+/// We're using [`KeyData`] internally, which is auto-incremented for each face.
```

Comment:
I think we should remove those two sentences completely. It's no longer relevant. A user should not care what is ID now.